### PR TITLE
Format imported numbers

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?177140

Wrote tests, then realized importer tests use a mock, not xlrd.

Also added link back to the beginning of the import wizard to this dead-end page.
screen shot 2015-07-27 at 3 25 24 pm

@benrudolph

### CODE REVIEW

File: corehq/apps/importer/templates/importer/partials/import_status.html [hunk 1]
```diff
@@ -59,6 +59,9 @@
         <script type="text/javascript">
             $("#export-download-status .loading-indicator").addClass('hide');
         </script>
+        {% if url %}
+            <a href="{{ url }}">Return to import cases</a>
```
Comment (czue) [inline corehq/apps/importer/templates/importer/partials/import_status.html]: missing `trans`

**Comment by czue** [comment]
:+1: from me. feel free to self merge after adding translation.

File: corehq/apps/importer/util.py [hunk 2]
```diff
@@ -121,6 +121,17 @@ def __init__(self, file_path, column_headers):
...
             self.has_errors = True
 
+    def _col_values(self, sheet, index):
+        return [self._fmt_value(cell) for cell in sheet.col(index)]
+
+    def _row_values(self, sheet, index):
+        return [self._fmt_value(cell) for cell in sheet.row(index)]
+
+    def _fmt_value(self, cell):
+        if cell.ctype == XL_CELL_NUMBER and int(cell.value) == cell.value:
```
Comment (benrudolph) [inline corehq/apps/importer/util.py]: i'm reading it as there's a cell that is a `XL_CELL_NUMBER` type and that the int version is equal to the regular version. doesn't that mean the cell is already an integer? or is this for converting decimals?

**Comment by benrudolph** [comment]
non-blocking comment, agree with cory on self merge

File: corehq/apps/importer/util.py (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/importer/util.py]: Yeah, the cell is already an integer, but xlrd treats all numbers like decimals: if the spreadsheet contains "1", the built-in `row_values` and `col_values` functions will return it as "1.0".

File: corehq/apps/importer/templates/importer/partials/import_status.html (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/importer/templates/importer/partials/import_status.html]: fixed, ty

**Pull request merged**